src/LiveQuery/Client.js

```diff
@@ -92,6 +96,12 @@ class Client {
           fields = this.subscriptionInfos.get(subscriptionId).fields;
         }
         response['object'] = this._toJSONWithFields(parseObjectJSON, fields);
+        if (typeof parseOriginalObjectJSON !== 'undefined') {
+          response['originalParseObject'] = this._toJSONWithFields(
```

Comment:
We use ‘original’ in the before/after hooks. Can you have a look? Would be nice to preserve semantics

Reply:
done